src/main/java/nextstep/subway/line/domain/Line.java

```diff
@@ -53,4 +55,145 @@ public String getColor() {
     public List<Section> getSections() {
         return sections;
     }
+
+    private void addSection(Section section) {
+        this.sections.add(section);
+    }
+
+    public void addSection(Station upStation, Station downStation, int distance) {
```

Comment:
```suggestion
    public void addSectionOf(Section section) {
```

많은 파라미터를 계속해서 가지고 다니기보다는, 해당 데이터를 가지고 있는 객체를 넘겨주는 것은 어떨까요?